### PR TITLE
Initial pass at adding submenus.

### PR DESCRIPTION
**Pull request by cmeiklejohn**
Remaining:
- [x] When submenu item is selected, the top-level item should also be.
- [x] Format of submenu should be reduced margin-top.

### CODE REVIEW

**Comment by cmeiklejohn** [comment]
Fixed the nav selection and highlighting, updated formatting, cleaned up presentation.

**Comment by cmeiklejohn** [comment]
@sebastianburckhardt If this is OK, I'll go ahead and merge it.

**Comment by sebastianburckhardt** [comment]
Just a sec... let me have a look

File: Sources/content/docs/interfaces.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    weight: 20
+---
+
+# Interfaces 
+
+The Reactive Machine interfaces: understanding how to build your application.
+
+* **Orchestrations** describe operations that are composed of one or more other operations. They are written in async/await style, and execute deterministically and reliably. When executing an operation, orchestrations can either perform it (meaning they wait for its completion) or fork it (meaning it executes completely independently of its parent).
+* **Activities** are operations that can be unreliable or nondeterministic, such as calls to external services.
+* *States** represent a small piece of information (cf. key-value pair, or a grain, or virtual actor) that can be atomically accessed via a specified set of read and update operations.
```
Comment (sebastianburckhardt) [inline Sources/content/docs/interfaces.md]: I think this should be **States**, not *States**

Comment (cmeiklejohn) [reply Sources/content/docs/interfaces.md]: That's definitely a typo.

Comment (cmeiklejohn) [reply Sources/content/docs/interfaces.md]: Fixed in 9f83499.

**Pull request merged**